middlewares/validators/fcmToken.js

```diff
@@ -0,0 +1,14 @@
+const joi = require("joi");
+
+export const fcmTokenValidator = async (req, res, next) => {
+  const schema = joi.object().strict().keys({
+    fcmToken: joi.string().required(),
```

Comment:
how are we validating the fcmToken passed. what if a malicious user tries to add some dummy fcmToken 👀

Reply:
![image](https://github.com/Real-Dev-Squad/website-backend/assets/61110378/00a218d7-5656-4a72-869b-38a3b48d7ffb)

will check in case if we do something in frontend to verify that